OSDOCS-7061: Documented the AWS Wavelength zone feature (#69537)
openshift/openshift-docs: modules/post-install-existing-local-zone-subnet.adoc

// Module included in the following assemblies:
//
// * post_installation_configuration/post-install-existing-local-zone-subnet.adoc
// * post_installation_configuration/post-install-existing-wavelength-zone-subnet.adoc

ifeval::["{context}" == "aws-compute-edge-tasks-local-zones"]
:local-zone:
endif::[]
ifeval::["{context}" == "aws-compute-edge-tasks-wavelength-zones"]
:wavelength-zone:
endif::[]

:_mod-docs-content-type: CONCEPT
[id="post-install-existing-local-zone-subnet_{context}"]
= Extend existing clusters to use AWS Local Zones

> If you want a Machine API to create an Amazon EC2 instance in a remote zone location, you must create a subnet in a Local Zone location. You can use any provisioning tool, such as Ansible or Terraform, to create subnets in the existing Virtual Private Cloud (VPC). You can configure the CloudFormation template to meet your requirements.

Reviewer: ```suggestion
If you want a Machine API to create an Amazon EC2 instance in a remote zone location, you must create a subnet in a Local Zone location. You can use any provisioning tool, such as Ansible or Terraform, to create subnets in the existing Virtual Private Cloud (VPC). 
```

> The following subsections include steps that use the AWS CloudFormation infrastructure to provision the network resoruces for your cluster.

Author: This suggestion is not as direct as the original.

> The following subsections include steps that use the AWS CloudFormation infrastructure to provision the network resoruces for your cluster.

Reviewer 2: I believe I was referring to something along the lines of this https://github.com/openshift/openshift-docs/pull/69537#discussion_r1484647638. Not sure where the suggestion came from.

> The following subsections include steps that use CloudFormation templates. Considering the limitation of NAT Gateways in AWS Local Zones, CloudFormation templates support only public subnets. You can reuse the template to create public subnets for each edge location to where you need to extend your cluster.

Reviewer: ```suggestion
The following subsections include steps that use CloudFormation to provision the network resources.
```

Author: Thanks. How about: 

> The following subsections include steps that use the AWS CloudFormation infrastructure to provision the network resoruces for your cluster.
